python/pyspark/ml/evaluation.py

```diff
@@ -328,6 +329,77 @@ def setParams(self, predictionCol="prediction", labelCol="label",
         kwargs = self._input_kwargs
         return self._set(**kwargs)
 
+
+@inherit_doc
+class ClusteringEvaluator(JavaEvaluator, HasPredictionCol, HasFeaturesCol,
+                          JavaMLReadable, JavaMLWritable):
+    """
+    .. note:: Experimental
+
+    Evaluator for Clustering results, which expects two input
+    columns: prediction and features.
+
+    >>> from pyspark.ml.linalg import Vectors
+    >>> scoreAndLabels = map(lambda x: (Vectors.dense(x[0]), x[1]),
```

Comment:
```scoreAndLabels``` -> ```featureAndPredictions```, the dataset here is different from other evaluators, we should use more accurate name. Thanks.